docs/content/commands/rclone_hashsum.md

```diff
@@ -37,13 +39,16 @@ rclone hashsum <hash> remote:path [flags]
 ## Options
 
 ```
-      --base64   Output base64 encoded hashsum
-  -h, --help     help for hashsum
+      --base64		Output base64 encoded hashsum
+      --download	Download the file and hash it locally; if this flag is not specified, the hash is requested from the remote
+  -h, --help		Help for hashsum
 ```
 
 See the [global flags page](/flags/) for global options not listed here.
 
 ## SEE ALSO
 
 * [rclone](/commands/rclone/)	 - Show help for rclone commands, flags and backends.
+* [rclone md5sum](/commands/rclone_md5sum/)	 - Shortcut equivalent to 'hashsum MD5'
```

Comment:
Did you edit this by hand? It is an auto generated file, generated by `make commanddocs`.

Or maybe you figured out some clever way of getting see also entries in with pflag - I don't see anything like that though.